session/object.go

```diff
@@ -0,0 +1,166 @@
+package session
+
+import (
+	"github.com/nspcc-dev/neofs-api-go/v2/session"
+	"github.com/nspcc-dev/neofs-sdk-go/object/address"
+)
+
+// ObjectContext represents NeoFS API v2-compatible
+// context of the object session.
+//
+// It is a wrapper over session.ObjectSessionContext
+// which allows abstracting from details of the message
+// structure.
+type ObjectContext session.ObjectSessionContext
+
+// NewObjectContext creates and returns blank ObjectContext.
+//
+// Defaults:
+//  - not bound to any operation;
+//  - nil object address.
+func NewObjectContext() *ObjectContext {
+	v2 := new(session.ObjectSessionContext)
+
+	return ObjectContextFromV2(v2)
+}
+
+// ObjectContextFromV2 wraps session.ObjectSessionContext
+// into ObjectContext.
+func ObjectContextFromV2(v *session.ObjectSessionContext) *ObjectContext {
```

Comment:
Why not `NewObjectContextFromV2` like in other places?

Reply:
copied from `ContainerSessionContext`

added commit that renames both